.travis.yml

```diff
@@ -12,6 +9,8 @@ matrix:
   fast_finish: true
   include:
     - php: '7.0'
+      env: DO_COVERAGE="y"
```

Comment:
The coverage should be done on every request.

Some BC trick can be coverage on some builds only.

Coveralls merges all result to have the total coverage.